compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -210,6 +238,10 @@ loco::TensorShape Algorithm::visit(const luci::CircleReshape *node)
     }
     if (unknown_dim_index != UINT32_MAX)
     {
+      if (input_element_count % output_element_count != 0)
```

Comment:
For negative tests purpose, should be moved to a separate PR?

Reply:
Before asking, how is this condition related with your issue?

Reply:
I can't catch the problem.

Reply:
It's not directly related with my change but I haven't better ideas to new negative tests (especially because we have in Reshape shape infer impl rather asserts instead of throwing exceptions).
But I see now, that it's better to move it to a separate PR ;)